src/Api/Mastodon/Account.php

```diff
@@ -52,37 +53,52 @@ class Account
 	var $fields = null;
 	/** @var bool|null */
 	var $bot = null;
+	/** @var bool */
+	var $group;
```

Comment:
Why do we use botz the @property annotation and the explicit creation of properties? I don't see a difference in the end (just in a different way).

Reply:
Mostly for autocompletion. Initially I wanted to separate the mandatory fields from the optional in the field declaration so that we can skip the optional fields we don’t support but I ended wondering why a field was missing from the create method.